frontend/src/styles/modules/_experiment-details.scss

```diff
@@ -251,6 +251,10 @@
       color: $transparent-black-7;
       font-size: $font-unit;
     }
+
+    figcaption img {
```

Comment:
@ckprice Two questions:

First, a minor sass syntactical question: why not just put this img block inside the above figcaption block?

```
     figcaption {
        color: $transparent-black-7;
        font-size: $font-unit;

        img {
          width: auto;
        }
      }
```

Second, why are you setting the width to auto? It looks like it was previously set to `36px`.